test/integration/agent/openflow_test.go

```diff
@@ -510,6 +524,28 @@ func checkConjunctionFlows(t *testing.T, ruleTable uint8, dropTable uint8, allow
 	}
 }
 
+func checkOVSFlowMetrics(t *testing.T, client ofClient.Client) {
+	expectedFlowCount := `
+	# HELP antrea_agent_ovs_flow_count [STABLE] Flow count for each OVS flow table. The TableID is used as a label.
+	# TYPE antrea_agent_ovs_flow_count gauge
+	`
+	tableStatus := client.GetFlowTableStatus()
+	totalFlowCount := 0
+	for _, table := range tableStatus {
+		expectedFlowCount = expectedFlowCount + fmt.Sprintf("antrea_agent_ovs_flow_count{table_id=\"%s\"} %s\n", strconv.Itoa(int(table.ID)), strconv.Itoa(int(table.FlowCount)))
```

Comment:
you can use `%d` and then you no longer need to the casts to `int` and the calls to `strconv.Itoa`

Reply:
same below

Reply:
Done. Missed this. Thanks.